exercises/Graphing-sexual-dimorphism-exploration-R.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: exercise
+topics: Graphing
+title: Sexual Dimorphism Exploration
+language: R
+---
+
+You are interested in understanding whether sexual size dimorphism is a general
+pattern in birds.
+
+Download and import a
+[large publicly available dataset of bird size measures](http://www.esapubs.org/archive/ecol/E088/096/avian_ssd_jan07.txt)
+created by [Lislevand et al. 2007](https://doi.org/10.1890/06-2054).
+
+Import the data into R. It is tab delimited so you'll want to use `sep = '\t'`
+as an optional argument when calling `read.csv()`. The `\t` is how we indicate a
+tab character to R (and most other programming languages).
+
+Using `ggplot`:
+
+1. Create a histogram of female masses (they are in the `F_mass` column). Change
+   the x axis label to "Female Mass(g)"
+2. A few really large masses dominate the histogram so create a log10 scaled
+   version. Change the x axis label to "Female Mass(g)" and the color of the
+   bars to blue (using the `fill = 'blue'` argument).
+3. Now let's add the data for male birds as well. Create a single graph with
+   histograms of both female and male body mass. Due to the way the data is
+   structured you'll need to add a 2nd `geom_histogram()` call that specifies a
+   new aesthetic).
```

Comment:
Unbalanced `)`. *As a style for the exercises I set hint or parenthetical info in italics.*